app/components/header/Header.js

```diff
@@ -1,6 +1,17 @@
 // @flow
 /* eslint react/no-set-state: 0 */
+import {
+  Button,
+  Collapse,
+  Form,
+  Input,
+  Nav,
+  Navbar,
+  NavbarToggler,
+  NavItem
+} from 'reactstrap';
 import React, { Component } from 'react';
+import type { Node, SyntheticEvent as Event } from 'react';
```

Comment:
type imports should go before relative imports. relative type imports should go after relative imports